src/pages/Home.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import Navigation from "../components/Navigation";
+import Logo from "../components/Logo";
+import Countries from "../components/Countries";
+
+const Home = () => {
+  return (
+    <div>
+      <Logo />
+      <Navigation />
+      <Countries />
+    </div>
```

Comment:
Create a new Layout component which will define the structure of the page (squelette). The end result will give something like 

```suggestion
    <Layout>
      <Logo />
      <Navigation />
      <Countries />
    </Layout>
```